docs/usage-as-end-user.md

```diff
@@ -0,0 +1,117 @@
+```
+kubectl port-forward -n kube-system deployment/hubble-ui 8080:12000
+```
+can be used to acess it locally.
+
+## `NetworkingConfig` options
+
+The `hubble.enabled` field describes whether hubble should be deployed into the cluster or not (default).
+
+The `debug` field describes whether you want to run cilium in debug mode or not (default), change this value to `true` to use debug mode.
+
+The `psp` field describes whether `cilium-operator` and `cilium-agent` shall be deployed with pod security policies or not (default).
+
+The `tunnel` field describes the encapsulation mode for communication between nodes. Possible values are `vxlan` (default), `geneve` or `disabled`.
+
+The `store` field describes which backend to use to store the identities. Can be either `etcd` (kvstore) or `kubernetes` (crd) (default).
+
+## Example `NetworkingConfig` manifest
+
+An example `NetworkingConfig` for the Cilium extension looks as follows:
+
+```yaml
+apiVersion: cilium.networking.extensions.gardener.cloud/v1alpha1
+kind: NetworkConfig
+hubble:
+  enabled: true
+#debug: false
+#psp: true
+#tunnel: vxlan
+#store: kubernetes
+```
+
+## Example `Shoot` manifest
+
+Please find below an example `Shoot` manifest with cilium networking configuration:
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: aws-cilium
+  namespace: garden-dev
+spec:
+  provider:
+    type: aws
+    infrastructureConfig:
+      apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
+      kind: InfrastructureConfig
+      networks:
+        vpc:
+          cidr: 10.250.0.0/16
+        zones:
+          - name: eu-west-1b
+            workers: 10.250.0.0/19
+            public: 10.250.32.0/20
+            internal: 10.250.48.0/20
+    controlPlaneConfig:
+      apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
+      kind: ControlPlaneConfig
+    workers:
+      - name: worker-w7xyu
+        minimum: 1
+        maximum: 2
+        maxSurge: 1
+        machine:
+          type: m5.large
+          image:
+            name: gardenlinux
+            version: 27.1.0
+        zones:
+          - eu-west-1b
+        volume:
+          type: gp2
+          size: 50Gi
+  networking:
+    type: cilium
+    providerConfig:
+      apiVersion: cilium.networking.extensions.gardener.cloud/v1alpha1
+      kind: NetworkConfig
+      hubble:
+        enabled: true
+    pods: 100.96.0.0/11
+    nodes: 10.250.0.0/16
+    services: 100.64.0.0/13
+  cloudProfileName: aws
+  secretBindingName: shoot-operator-aws
+  region: eu-west-1
+  purpose: evaluation
+  kubernetes:
+    version: 1.18.5
+  addons:
+    kubernetesDashboard:
+      enabled: false
+    nginxIngress:
+      enabled: false
+  maintenance:
+    timeWindow:
+      begin: 000000+0000
+      end: 010000+0000
+    autoUpdate:
+      kubernetesVersion: true
+      machineImageVersion: true
+  hibernation:
+    schedules:
+      - start: '00 17 * * 1,2,3,4,5'
+        location: Europe/Berlin
```

Comment:
Hm, generally, this is hard to maintain for a non-provider extension. What about only proposing the `networking` fields?

```suggestion
  networking:
    type: cilium
    providerConfig:
      apiVersion: cilium.networking.extensions.gardener.cloud/v1alpha1
      kind: NetworkConfig
      hubble:
        enabled: true
    pods: 100.96.0.0/11
    nodes: 10.250.0.0/16
    services: 100.64.0.0/13
  ...
```

Reply:
If the provider configs are skipped here, I would add something like
> If you would like to see provider specific shoot example, please check out the documentation of the well know extensions, list of which can be found [here](https://github.com/gardener/gardener/tree/master/extensions#infrastructure-provider).